api-client/src/utils/GeoJson.ts

```diff
@@ -0,0 +1,22 @@
+import { Feature, FeatureCollection } from 'geojson'
+import LocationModel from '../models/LocationModel'
+
+export const convertToPoint = (location: LocationModel): Feature => {
+  return {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [Number(location._latitude), Number(location.longitude)]
```

Comment:
```suggestion
      coordinates: [Number(location.latitude), Number(location.longitude)]
```

I think it is first long and then lat. But im not sure.

Reply:
Yep, longitude seems to come first: https://stackoverflow.com/questions/19098667/order-of-coordinates-in-geojson#57653830

Reply:
Where can I upvote your answer 😉?